translations/ui-licenses/en.json

```diff
@@ -81,6 +81,16 @@
   "section.terms": "Terms",
 
   "settings.general": "General",
+  "settings.general.message": "These are your general app settings.",
+  "settings.some-feature": "Some Feature",
+  "settings.some-feature.message": "These are your settings for some app feature.",
+  "settings.pickLists": "Pick lists",
```

Comment:
Let's try and keep the keys for this feature scoped a bit.  Like how all the organizations keys start with `organizations.`, or the amendments start with `amendments.`. 

If you feel some are extremely generic all will definitely see reuse, then move them to the top of the file.